generator/Can_Library_c.py

```diff
@@ -1,42 +1,46 @@
-"""
+'''
 Generate Can_Libary.c file.
 Run this file (with the spec path as a command line argument) to write just
 Can_Libary.c or main.py to write all files.
-"""
+'''
 import sys
-sys.path.append("ParseCAN")
+sys.path.append('ParseCAN')
```

Comment:
Glad you're fixing this, I knew I was inconsistent on quotes but never got around to fixing them...